readme.txt

```diff
@@ -59,94 +59,6 @@ If you have bugs to report, please go to [the plugin's GitHub repository](https:
-
-= 1.9.2 =
-
-* Fix missing autoloader
-
-= 1.9 =
-
-* Introduces a new option to the plugin: adding a comment policy was never easier than this: just create a comment
-policy page, toggle the option on and select it, and commenters will have to accept your comment policy before being
-able to comment.
-* Fixes a bug where editing a comment on the quick edit screen would cause that comment to lose its parent.
-* Enhances performance by preventing too frequent option updates.
-* Remove all direct DB queries in favor of using WordPress core functions.
-
-= 1.8.1 =
-
-* Fixed a couple of PHP 7.4 related issues.
-
-= 1.8 =
-
-* Changed namespace to `JoostBlog`.
-* Removed Yoast branding.
-* Updated plugin to require PHP 7.4.
-
-= 1.7 =
-
-* Bugfixes:
-    * Fixed: the "Email commenters" link would not be displayed in the WordPress admin bar and in the Comments list.
-    * Fixed: the "Email commenters" link in the front end admin bar wouldn't work when jQuery wasn't enqueued.
-    * Fixed: the notification emails for new comments would have incorrect content for the Author line and the text displayed before the comment.
-
-= 1.6 =
-
-* Fix language packs.
-
-= 1.5 =
-
-* Bugfixes:
-    * Fixed: comment recipient dropdown would reset on reload of the page.
-    * Fixed: admin bar CSS showing when no admin bar is showing.
-
-= 1.4 =
-
-* Enhancements:
-    * Limit the roles shown in the comment notifications dropdown to roles that normally exist and can write. Introduces a new filter to allow expanding them.
-
-= 1.3 =
-
-* Enhancements:
-    * Add option to restrict comments that are too long, next to too small.
-    * Add `reply-to` header to comment notification and moderation emails, pointing to the post author.
-    * Preserve the active tab when saving settings.
-    * Remove `[...]` from pingback / trackback excerpt in cleaned emails as that's already included by core.
-    * Replace link to ARIN with link to [ip-lookup.net](http://ip-lookup.net) for details about the IP.
-    * Refactored code for readability and code quality.
-    * Refactor upgrade routine to do less DB queries.
-
-= 1.2 =
-
-Fixes several issues:
-
-* Differentiate between AJAX request and normal POST for nonce checking, fixes #7.
-* Make sure comment type isn't empty in cleaned email, fixes #8.
-* Allow setting the comment parent to 0, fixes #10.
-* Prevents defaults from being reinstated, fixes #14.
-
-Also:
-
-* Adds translator comments to all strings with `sprintf` / `printf`.
-* Updates to new version of Yoast i18n, in the process switching from `translate.yoast.com` to `translate.wordpress.org` and removing packaged translations.
-* Added `yarn.lock` and removed no longer needed i18n grunt tasks.
-
-= 1.1.1 =
-
-* Add text domain so the plugin can be translated.
-
-= 1.1 =
-
-* Add comment routing option, adds a dropdown in a post's discussion settings, allowing the routing of comment emails to another user.
-
 = 1.0 =
 
 * Initial version.
```

Comment:
Maybe note here:

> The initial version is based on the plugin formerly known as Yoast Comment Hacks to get rid of branding since Yoast no longer owns the plugin.

Reply:
Done :+1: